frontend/src/components/elements/Markdown/Markdown.tsx

```diff
@@ -33,11 +38,25 @@ export default function Markdown({
   const styleProp = { width }
   return (
     <div className="stMarkdown" style={styleProp}>
-      <StreamlitMarkdown
-        isCaption={element.isCaption}
-        source={element.body}
-        allowHTML={element.allowHtml}
-      />
+      {element.help ? (
+        <StyledLabelHelpWrapper>
+          <StreamlitMarkdown
+            isCaption={element.isCaption}
+            source={element.body}
+            allowHTML={element.allowHtml}
+          />
+          <InlineTooltipIcon
+            content={element.help}
+            isLatex={element.elementType === 4}
```

Comment:
nit: Can we somehow use the constant from the generated proto class here? I think the `Type.LATEX` constant is generated as well (but not 100% sure).

Reply:
Great advice, it's done 👍